client/src/components/UserDashBoard.jsx

```diff
@@ -1,16 +1,121 @@
 const React = require('react');
 const UserDashBoardKeywordsEntryBar = require('./UserDashBoardKeywordsEntryBar.jsx');
+const UserDashBoardMonitoredWordResult = require('./UserDashBoardMonitoredWordResult.jsx');
 
 class UserDashBoard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: true,
+      monitoringResults: this.props.monitoringResults,
+      errorMessage: ''
+    };
+
+    this.handleAddMonitoredWords = this.handleAddMonitoredWords.bind(this);
+    this.handleRemoveMonitoredWords = this.handleRemoveMonitoredWords.bind(this);
+  }
+  componentDidMount() { 
+    // Retrieve user monitored keywords and associated bills to populate the dashboard
+    $.get('user/' + this.props.username + '/keywords')
+      .done(data => this.setState({
+        isFetching: false,
+        monitoringResults: data,
```

Comment:
can you explain the rationale behind this particular deviation from the agreed format?
shouldn't it be [{word: '<monitoredwords>', associatedKeywords: [...,...,...], relatedBills['id1','id2'}, {word: '<monitoredwords2>, associatedKeywords: [...,...,...], relatedBills['id3','id5'}, {word: ]